packages/nx-plugin/package.json

```diff
@@ -4,5 +4,11 @@
   "version": "0.0.1",
   "type": "commonjs",
   "generators": "./generators.json",
-  "executors": "./executors.json"
+  "executors": "./executors.json",
+  "dependencies": {
+    "semver": "^7.5.2",
+    "@nx/devkit": "^16.1.4",
+    "@nx/storybook": "^16.1.4",
+    "@nx/js": "16.1.4"
+  }
```

Comment:
Is there sense to lint private packages for deps?

Reply:
I can't see why it would hurt. The intention in the future is to publish this so that others can also build their own repo